test/hts-precompile/erc-20/IERC20.js

```diff
@@ -33,24 +33,24 @@ describe('IERC20 Test Suite', function () {
   let tokenAddress;
   let IERC20;
   let signers;
-  const TOTAL_SUPPLY = 1000;
-  const AMOUNT = 33;
+  const TOTAL_SUPPLY = BigInt(1000);
+  const AMOUNT = BigInt(33);
```

Comment:
Just a detail, For `BigInt` literals, you might want to use the `n` suffix, _i.e._ `1000n`.